charts/generic-aws-prometheus-alerts/values.yaml

```diff
@@ -32,10 +32,10 @@ rdsAlertsCPUThresholdMinutes: 2
 # https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/CHAP_Limits.html#RDS_Limits.MaxConnections then provides
 # calculations for each database on the maximum number of database connections so for a db.t3.small database it will be
 # LEAST({2Gib/9531392}, 5000) = LEAST({2 * 1024 * 1024 * 1024/9531392}, 5000) = LEAST(225, 5000) = 225.
-rdsAlertsConnectionThreshold: 180
+rdsAlertsConnectionThreshold: 5
```

Comment:
why these connection count are low.?

Reply:
To trigger the alerts, it was set to 5. Will revert this once the alert trigger is tested

Reply:
this is generic alert and test alert values should be changed to terraform hmpps-service-catalogue (dev namespace) deployment